app/assets/javascripts/state/SavedAnnotations.ts

```
@@ -43,6 +43,11 @@ class SavedAnnotationState extends State {
     private async fetch(id: number): Promise<SavedAnnotation> {
         const url = `${URL}/${id}.json`;
         const response = await fetch(url);
+
+        if (response.status === 403) {
+            return undefined;
+        }
```

This is a bit odd as it would now fail silently? We should at least log something to the console as this shouldn't happen.

---

The 403 error is still shown in the console, but the error parsing `response.json()` is avoided.

But can leave this out as the other changes fixes the whole issue